sto/ethereum/linking.py

```diff
@@ -0,0 +1,182 @@
+import re
```

Comment:
[Can be ignored] lifted from populus, needed to link bytecode for contracts dependent on libraries.